src/Lifi.ts

```diff
@@ -237,6 +303,12 @@ class LIFI {
     return balances.getTokenBalance(walletAddress, token)
   }
 
+  /**
+   * Returns the balances for a list tokens a wallet holds across all chains we have aggergated.
```

Comment:
```suggestion
   * Returns the balances for a list tokens a wallet holds across all chains we have aggregated.
```

Reply:
Maybe "we" is also a bit to informal here. "Across all aggregated chains" maybe?